frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-details.tsx

```diff
@@ -66,8 +66,8 @@ const OperatorHubDetails: React.FC<OperatorHubDetailsProps> = ({ obj: operatorHu
                 })}
             </DetailsItem>
           </DescriptionList>
-        </div>
-      </div>
+        </GridItem>
+      </Grid>
```

Comment:
OperatorHub details page, the left side are too close with right column so that the right side looks smaller
<img width="1262" alt="OperatorHub-details-PR" src="https://github.com/user-attachments/assets/01a15fff-62e1-45eb-acce-9ad40a4cf154" />


For comparison - latest 419 nightly
<img width="1234" alt="Screenshot 2025-05-06 at 4 36 01 PM" src="https://github.com/user-attachments/assets/90943eb3-ca1a-4db9-8417-990130538a0a" />

Reply:
It seems this is expected if we switch to `GridItem`, is this correct? 
This issue is quite obvious on OperatorHub detail page then other workloads pages

Reply:
looks like I missed `hasGutter` for this page too:

![Uploading image.png…]()